packages/webview_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## NEXT
+
+* Remove unnecessary `dart:typed_data` import.
+* Annotate `override` to overriden member.
```

Comment:
nit:
```suggestion
* Annotate overridden member with `@override`.
```

(Actually a patch note doesn't need to be this specific and can just be something like "Resolve linter warnings.")

Reply:
Okay, I'll use "Resolve linter warnings".